src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsWindowMapping.java

```diff
@@ -46,19 +52,30 @@ public static HearingWindow buildHearingWindow(@Valid SscsCaseData caseData) {
     }
 
     public static LocalDate getHearingWindowStart(@Valid SscsCaseData caseData) {
-        if (isCasePostponed(caseData)) {
+        if (isCaseAdjourned(caseData)) {
+            // TODO add to "Other" field time details from adjournCaseTime
```

Comment:
Some info logging would be a huge help in tracing how the dates have been resolved

Reply:
done